src/viam/components/camera/camera.py

```diff
@@ -43,6 +43,17 @@ async def get_image(
         NOTE: If the mime type is ``image/vnd.viam.dep`` you can use :func:`viam.media.video.ViamImage.bytes_to_depth_array`
         to convert the data to a standard representation.
 
+        ::
+
+            my_camera = Camera.from_robot(robot=robot, name="my_camera")
+
+            # Assume "frame" has a mime_type of "image/vnd.viam.dep"
+            frame = await my_camera.get_image()
```

Comment:
Perhaps add this as the mime type as part of the request? I think that would showcase the method better

```python3
frame = await my_camera.get_image(mime_type = CameraMimeType.VIAM_RAW_DEPTH)
```